### ISSUE TITLE
booking confirmation message not sent

### ISSUE THREAD

**Issue by asafrot** [wootwoot1234/Superhost-Tools#5]
Hi Tom, 

when a booking is made superhost tools suppose to send a confirmation message to the guest but unfortunately it doesn't

thanks

Asaf

**Comment by wootwoot1234** [comment]
Hmmmm. Let me take a look.  What email are you using to sign into superhosttools.com?

**Comment by asafrot** [comment]
This email
adam.klein.com@gmail.com

On Aug 22, 2017 00:41, "Tom" <notifications@github.com> wrote:

Hmmmm. Let me take a look. What email are you using to sign into
superhosttools.com?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/wootwoot1234/Superhost-Tools/issues/5#issuecomment-323760767>,
or mute the thread

**Comment by wootwoot1234** [comment]
Which listing had the issue?  Can you tell me when the booking occurred (same day, 30 days in advance)?  Have all your booking messages not been working or just for one guest?  I have book confirmation messages setup on my account and they work fine.  Maybe there is something unique about your case that's causing the bug?  Thanks for the help in figuring this out.

**Comment by asafrot** [comment]
All listings. All bookings. the bookings never took place on the same day.
the other messages work 90% sometimes they don't as it fails to log in.
After every booking not only it doesn't send booking confirmation a few
hours later I get an email from superhost tools - Airbnb Login Failed

SuperhostTools.com


There was a problem logging into your Airbnb account. Please update
your credentials as soon as possible to avoid any interruptions in
your service.


Sent with care from SuperhostTools.com
<http://email.mail.superhosttools.com/c/eJw9jUEOgyAQRU-jSzIMCOOChbbhAu0FELCaohil969t0iZ_8fIX7wUTW41jPRsEroGQgwaNgjWSU8uk7a9CqJ6rC6Lt-krC4ubEjtcW9ykfpeScDubzUk8GBueawQvSFCS0BN4HCKQ0SR752NTJTKVslegqtOduP8n9L9mNC25hzxTn9XOcvcc3ePIbNIMz9A>

**Comment by wootwoot1234** [comment]
Hi Asafrot,

Can you let me know ASAP next time you get a booking (with the time of the booking) so I can look at the server logs and see what's going on?

This is a tough bug to solve because I'm not seeing it with my account.  I have booking confirmation messages and they work fine.  I really appreciate your help figuring this out.

As for you getting locked out of your account, I'm not sure why that's happening.  I get an email whenever it happens to a user so I'm aware of the issue.  I don't know if it's because of all the calls to the airbnb servers per account or if it's because superhosttools.com is servicing so many accounts now.  It might be because you have more listings than most?  I'm not sure.  I'm going to change the server settings to update the prices every 24 hours instead of every 6 hours.  This will greatly reduce the calls to Airbnb.com and may help this issue.

**Comment by asafrot** [comment]
When you ready I can book one night from another account I have and you'll
see for yourself.

I have 4 listings is that a lot?

I have another tool, pricing labs, that log into Airbnb - maybe there's a
clash?

**Comment by wootwoot1234** [comment]
Thanks for helping me figure out these issues. 

That is more listing than most but no, it's not a ton. However, each listing adds more calls to Airbnb's servers so if Airbnb is locking out users based on how many calls it gets from services like superhost tools your account might get locked out more often. 

Are you being required to login again to Airbnb in pricelabs too?

I'm going to look over the code and see if I can come up with a way to make less calls to Airbnb's servers. 

As for the booking confirmation message. If you want to make a booking just let me know when so I can look at the logs. 

Thanks again for your help.

**Comment by wootwoot1234** [comment]
Let's continue the discussion about your account getting locked out and not being able to log back in here. 

https://github.com/wootwoot1234/Superhost-Tools/issues/9

That way we can brainstorm together.

**Comment by asafrot** [comment]
Hi Tom,

that problem only happened once with pricing lab since I started using
superhost tools.
I'm going to book now and see if it happens again

Asaf

**Comment by wootwoot1234** [comment]
Did the booking confirmation message go out?

**Comment by asafrot** [comment]
I've just booked penthouse oasis - let's see what happens

**Comment by wootwoot1234** [comment]
Ok, it can take up to 15 min.

**Comment by asafrot** [comment]
It's been 30min and no message.
In few hours I'll get an email if failed log in

**Comment by asafrot** [comment]
another booking just took place now. no confirmation message was sent
although I can see the booking is in the system of superhost tools

**Comment by asafrot** [comment]
Airbnb reported frequent suspicious logins from the US by Mozilla until
they blocked it and I had to change the password for airbnb. I also removed
iPhone devices that tried to log in to my account. a few minutes later
superhost tools reported failed log in. I updated the password there

**Comment by asafrot** [comment]
and in superhost tools it failed to login

**Comment by wootwoot1234** [comment]
Sorry for not getting back to you sooner, I'm currently traveling and am not at my computer all the time.  :)

I took a look at the logs and I'm not seeing any errors but it's interesting that you get logged out when you get a new booking.  My current theory is that maybe airbnb doesn't like it that superhost tools pings their servers too often (maybe it looks spammy to them?).  I added some code that will spread out the calls to their servers and also only check for new reservations every 20 min (instead of 15 min).  Fingers crossed, this will fix the issue.  I'll keep an eye on the logs and see if you get logged out again.
